### ISSUE TITLE
socket hung up error while deployment with snarkyjs v0.4.1

### ISSUE THREAD

**Issue by ashish173** [o1-labs/snarkyjs#256]
I am trying to deploy the sudoku example application on berkeley network but got an error of socket hang up. This happened right after the `Build transaction` step in the deployment sequence.
Trivo (from discord#zapp-developers) helped to solve the issue by changing the snarkyjs version from 0.4.1 to 0.3.7 and it worked.

Below is the error that I got in console as soon as I confirmed to send the transaction. 
```
Are you sure you want to send (yes/no)? · yes
✔ Send to network
  Failed to send transaction to relayer. Errors: request to http://devnet-producer_coda_1:3085/graphql failed, reason: socket hang up
```

## zk system         
```
Please include the following when submitting a Github issue:

  System:
    OS: macOS 11.6.5
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
  Binaries:
    Node: 16.15.1 - /usr/local/opt/node@16/bin/node
    Yarn: 1.7.0 - /usr/local/bin/yarn
    npm: 8.11.0 - /usr/local/opt/node@16/bin/npm
  npmPackages:
    snarkyjs: ^0.4.1 => 0.4.1 
  npmGlobalPackages:
    zkapp-cli: 0.4.8
```

**Comment by mitschabaude** [comment]
this should be fixed in the new version `0.4.3`, which we just published!

**Comment by mitschabaude** [comment]
(note: new CLI version will be published later today)
